docs/core/run-time-config/compilation.md

```diff
@@ -125,3 +125,13 @@ Project file:
 
 </Project>
 ```
+
+## ReadyToRun
+
+- Configures whether the .NET Core runtime uses pre-compiled images for framework code (and user assemblies, if pre-compiled images are available). Disabling this option forces the runtime to JIT-compile framework code.
+- For more information see [ReadyToRun](../whats-new/dotnet-core-3-0.md#readytorun-images).
+- Default: Enabled (`1`).
+
+| | Setting name | Values |
+| - | - | - |
+| **Environment variable** | `COMPlus_ReadyToRun` | `1` - enabled<br/>`0` - disabled |
```

Comment:
Can we clarify that `1` is the default?

Reply:
It is clarified in the bulleted list. Is that okay? That's how we've done it for the other options too.

Reply:
Ah! didn't notice :). Thanks